integration-tests/docker/druid.sh

```diff
@@ -73,4 +75,23 @@ setupConfig() {
       var=$(echo "$evar" | sed -e 's?^\([^=]*\)=.*?\1?g' -e 's?_?.?g')
       setKey $DRUID_SERVICE "$var" "$val"
   done
-}
\ No newline at end of file
+}
+
+setupData()
+{
+  # The "query" and "security" test groups require data to be setup before running the tests.
+  # In particular, they requires segments to be download from a pre-existing s3 bucket.
+  # This is done by using the loadSpec put into metadatastore and s3 credientials set below.
+  if [ "$DRUID_INTEGRATION_TEST_GROUP" = "query" ] || [ "$DRUID_INTEGRATION_TEST_GROUP" = "security" ]; then
```

Comment:
Hmm, this doesn't necessarily need to be changed now, but I think longer term we are going to need a mapping of test groups to configurations, but I'm not sure it should be encoded in a file in the container. This part should maybe be repurposed to allow running of initialization scripts that configurations bring with them to the container, and this stuff be in a script that initializes the legacy integration test environment.

Reply:
I think that's a good idea. Should definitely revisit this